themes/le-2025/assets/css/le-2025-theme-input.css

```diff
@@ -487,62 +487,92 @@
   .hero-bg {
-    background-image: url('/images/radiant_london_resized/radiant_london_5_1080w.jpg');
+    background-image: image-set(
+      url('/images/radiant_london_resized/radiant_london_5_1080w.webp') type("image/webp"),
+      url('/images/radiant_london_resized/radiant_london_5_1080w.jpg') type("image/jpeg")
+    );
   }
 }
 
 @screen xl {
   .hero-bg {
-    background-image: url('/images/radiant_london_resized/radiant_london_5_1280w.jpg');
+    background-image: image-set(
+      url('/images/radiant_london_resized/radiant_london_5_1280w.webp') type("image/webp"),
+      url('/images/radiant_london_resized/radiant_london_5_1280w.jpg') type("image/jpeg")
+    );
   }
 }
 
 @screen 2xl {
   .hero-bg {
-    background-image: url('/images/radiant_london_resized/radiant_london_5_1920w.jpg');
+    background-image: image-set(
+      url('/images/radiant_london_resized/radiant_london_5_1920w.webp') type("image/webp"),
+      url('/images/radiant_london_resized/radiant_london_5_1920w.jpg') type("image/jpeg")
+    );
   }
 }
 
 @media (min-width: 480px) and (min-resolution: 2dppx) {
   .hero-bg {
-    background-image: url('/images/radiant_london_resized/radiant_london_5_480w@2x.jpg');
+    background-image: image-set(
+      url('/images/radiant_london_resized/radiant_london_5_480w@2x.webp') type("image/webp"),
+      url('/images/radiant_london_resized/radiant_london_5_480w@2x.jpg') type("image/jpeg")
+    );
   }
 }
 
 @media (min-width: 640px) and (min-resolution: 2dppx) {
   .hero-bg {
-    background-image: url('/images/radiant_london_resized/radiant_london_5_640w@2x.jpg');
+    background-image: image-set(
+      url('/images/radiant_london_resized/radiant_london_5_640w@2x.webp') type("image/webp"),
+      url('/images/radiant_london_resized/radiant_london_5_640w@2x.jpg') type("image/jpeg")
+    );
     background-position: top center;
   }
 }
 
 @media (min-width: 1080px) and (min-resolution: 2dppx) {
   .hero-bg {
-    background-image: url('/images/radiant_london_resized/radiant_london_5_1080w@2x.jpg');
+    background-image: image-set(
+      url('/images/radiant_london_resized/radiant_london_5_1080w@2x.webp') type("image/webp"),
+      url('/images/radiant_london_resized/radiant_london_5_1080w@2x.jpg') type("image/jpeg")
+    );
   }
 }
 
 @media (min-width: 1280px) and (min-resolution: 2dppx) {
   .hero-bg {
-    background-image: url('/images/radiant_london_resized/radiant_london_5_1280w@2x.jpg');
+    background-image: image-set(
+      url('/images/radiant_london_resized/radiant_london_5_1280w@2x.webp') type("image/webp"),
+      url('/images/radiant_london_resized/radiant_london_5_1280w@2x.jpg') type("image/jpeg")
+    );
   }
 }
 
 @media (min-width: 1920px) and (min-resolution: 2dppx) {
   .hero-bg {
-    background-image: url('/images/radiant_london_resized/radiant_london_5_1920w@2x.jpg');
+    background-image: image-set(
+      url('/images/radiant_london_resized/radiant_london_5_1920w@2x.webp') type("image/webp"),
```

Comment:
Are you adding webp options because they compress better and thus reduce download size?

A while ago I did a comparison of jpeg and webp compression and found it to be a wash, and to the extent that webp sometimes appeared to produce smaller images it also did so at lower quality.

Reply:
You’re right, in this case it is unnecessary.